generate_docs.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+
+# this script generates auto-generated documentation for the Text_extensions for pandas package
+# using Sphinx apidoc and Sphinx autodoc 
+# 
+# To make adjustments to the landing page, edit the index.rst file in the docs directory 
+# To make adjustments to the style or change settings, modify the conf.py in the docs directory 
+# 
+# This script is made to run using the same python environment as is used for the package
+# and should be run accordingly 
+
+
+#install sphinx if it is not already installed
+conda install -y sphinx
```

Comment:
Now that I've merged #38, can you please remove this line from here and add `sphinx` to the dependencies in `config/dev_reqs.txt`?